### PR TITLE
Progress_Bar 

### PR DESCRIPTION
**Pull request by adrianpfeiffer**
added nice progress bar
-> can be used in "Landingpage" 

Important: 
AdMob überdeckt den "Privacy Policy" Button
-> Ich würde AdMob entfernen für Webapp (wenn das nicht eh schon so ist), dann wäre dieser pr richtig & richtig aligned. Wenn wir uns dagegen entscheiden, muss ich nochmal ran (Aufwand: low)

### CODE REVIEW

**Comment by adrianpfeiffer** [comment]
fuck pls check last commit! There was an (easy) issue to be resolved and i merged branch. Dunno if its correct

**Comment by abegehr** [comment]
@adrianpfeiffer, I do not understand this: https://github.com/abegehr/monsterjump/pull/55#issuecomment-605525253

**Comment by abegehr** [comment]
@adrianpfeiffer, 'AdMob überdeckt den "Privacy Policy" Button' – kannst Du bitte einen screenshot teilen?

File: lib/overlays/menu_overlay.dart [hunk 1]
```diff
@@ -73,7 +75,11 @@ class MenuOverlay extends StatelessWidget {
                 }),
             // PrivacyPolicyButton
```
Comment (abegehr) [inline lib/overlays/menu_overlay.dart]: this comment should be above the privacy policy button

**Comment by abegehr** [comment]
@adrianpfeiffer is this still current or can be closed?

**Comment by adrianpfeiffer** [comment]
Screenshot 2020-04-06 at 20 04 33

screenshot from web. On Mobile admob is just above impressum button

**Comment by adrianpfeiffer** [comment]
> @adrianpfeiffer is this still current or can be closed?

yes can be closed

File: ios/Runner/Base.lproj/Main.storyboard [hunk 2]
```diff
@@ -1,8 +1,10 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (abegehr) [inline ios/Runner/Base.lproj/Main.storyboard:1]: why is a ios/ file changed for a coronajump PR?

**Comment by abegehr** [comment]
closing https://github.com/abegehr/monsterjump/pull/55#issuecomment-609955297

**Pull request closed without merging**